db/fixtures/001_refresh.rb

```diff
@@ -0,0 +1,3 @@
+# fix any bust caches post initial migration
+ActiveRecord::Base.send(:subclasses).each { |m| m.reset_column_information }
+SiteSetting.refresh!
```

Comment:
Those 2 lines is the only remaining code from `db/fixtures/001_categories.rb`. I guess it's still needed?

Reply:
I think both are still needed, yes.